pkg/monitor/api.go

```diff
@@ -10,6 +10,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
```

Comment:
nit: move to bottom :trollface: 

Also somebody should set `goimport` for David